Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1287 @@
+      description: A single IPv4 host address or CIDR or a comma-separated list of
+        IPv4 host addresses or CIDRs to update.
+      isArray: true
+      name: host_address
+      required: true
+      secret: false
+    - default: false
+      description: A single description that applies to all of the specified hosts
+        or a comma-separated list of descriptions, each of which applies to a specific
+        host.
+      isArray: true
+      name: annotation
+      required: false
+      secret: false
+    deprecated: false
+    description: Replaces all the hosts on the outbound whitelisted list.
+    execution: false
+    name: na-ed-outbound-whitelisted-hosts-replace
+    outputs:
+    - contextPath: NetscoutAED.OutboundWhitelistHost.annotation
+      description: A description of the host.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.host_address
+      description: IPv4 host address or CIDRs.
+      type: String
+    - contextPath: NetscoutAED.OutboundWhitelistHost.update_time
+      description: The time the host was last updated/set
+      type: Date
+  - arguments:
+    - default: false
+      description: List of protection group IDs.
+      isArray: false
+      name: pgid
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'true'
+      description: 'Set the protection group mode to active (true) or inactive (false).
+        Default: true.'
+      isArray: false
+      name: active
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: low
+      description: 'The protection level (None = use the global protection level,
+        low, medium, high). Default: low'
+      isArray: false
+      name: protection_level
+      predefined:
+      - low
+      - medium
+      - high
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Turn traffic profiling on (true) or off (false) for one or more
+        of the protection groups.
+      isArray: false
+      name: profiling
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specify the number of days, from 1 to 14, over which profiling
+        will run. Required when profiling is set to true.
+      isArray: false
+      name: profiling_duration
+      predefined:
+      - '1'
+      - '2'
+      - '3'
+      - '4'
+      - '5'
+      - '6'
+      - '7'
+      - '8'
+      - '9'
+      - '10'
+      - '11'
+      - '12'
+      - '13'
+      - '14'
+      required: false
+      secret: false
+    deprecated: false
+    description: Updates the settings for one or more protection groups.
+    execution: false
+    name: na-ed-protection-groups-update
+    outputs:
+    - contextPath: NetscoutAED.Protection_Group.active
```

Comment:
Since you changed this path in the code, you should change here respectively. 
Also don't forget to re-generate the README due to this change.

Reply:
Done